src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget/StakingInput/StakingInput.tsx

```diff
@@ -7,10 +7,13 @@ import { Action, ActionTypes } from '~redux';
 import { mapPayload } from '~utils/actions';
 import { useAppContext, useColonyContext } from '~hooks';
 
-import StakingControls from './StakingControls';
-import StakingSliderDescription from './StakingSliderDescription';
-import StakingSliderAnnotation from './StakingSliderAnnotation';
-import StakingWidgetSlider from './StakingWidgetSlider';
+import { useStakingWidgetContext, getStakeFromSlider } from '..';
+import {
```

Comment:
I noticed this is a different import style which I like since its more concise. Do we want to incorporate this style everywhere going forward?